eng/Build.props

```diff
@@ -6,4 +6,11 @@
     <ProjectToBuild Include="$(RepoRoot)src/VirtualMonoRepo/InitializeVMR.proj" BuildInParallel="false" />
   </ItemGroup>
 
+  <!-- Do not populate ProjectToBuild except when we are explicitly in a non-default pass (pass 1, DotNetBuildPass == '')
+       so that we get the arcade defaults for what to build. -->
+  <ItemGroup Condition="'$(DotNetBuildPass)' != ''">
```

Comment:
After @jkoritzinsky's change propagates to the VMR, this should be no longer necessary